sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosRegionSwitchHint.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
```

Comment:
I would not put this in the models package - ThroughputRetryOptions, DirectConnectionConfig etc. are all in the parent package - to me this belongs in the same package as those two?

Reply:
Fixed in next iteration.